### PR TITLE
Update user.py

### LINKED ISSUE #249: [Typos][Set2]Fix Minor typos in models/user.py
**Describe the bug**
The `models/user.py` has 2 typos that need fixing.
* https://github.com/systers/mentorship-backend/blob/develop/app/database/models/user.py#L60 `extra `"` in the docstring should be 3, not 4
* https://github.com/systers/mentorship-backend/blob/develop/app/database/models/user.py#L112 `should be `user name is` not` id`

**Expected result**
All the above typos to be fixed in the `user.py` as a single commit.

**Estimated time**
1/2 hour

### PR DESCRIPTION
**Pull request by Xu-Zijun**
The models/user.py has 2 typos that I supposingly fixed.

On line 60, I removed the extra "
On line 112, I removed the extra id.

Fixes #249

### CODE REVIEW

**Review by annabauza — APPROVED** [review]
Perfect!

File: app/database/models/user.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def json(self):
 
     def __repr__(self):
         """Returns the user's name and username. """
-        return "User name id %s. Username is %s ." % (self.name, self.username)
+        return "User name %s. Username is %s ." % (self.name, self.username)
```
Comment (isabelcosta) [inline app/database/models/user.py:112]: Very nice catch! 👏  @Xu-Zijun

**Review by isabelcosta — APPROVED** [review]
Thank you so much for your contribution @Xu-Zijun 🎉

**Comment by isabelcosta** [comment]
@Xu-Zijun We have a current contest going on with Google Code-in and because we have a problem of a non persistent sqlite database (that get erased when a new merge is don), I'll avoid merging PRs for now. Steps are being taken to fix the non persistent database, so as soon as I can I'll merge this PR.

**Comment by annabauza** [comment]
@isabelcosta Xu is part of gci 😊

**Comment by annabauza** [comment]
https://codein.withgoogle.com/dashboard/task-instances/5985114249494528/

**Comment by isabelcosta** [comment]
@annabauza okay! I assumed that Xu wasn't because the PR is not targeting `gci-dev` branch. I tried changing, but it warned me about showing commits from other branches. So I stopped.
@Xu-Zijun for next contributions regarding GCI please sent them to the `gci-dev` branch.

You can click the edit button for this one, or for next PRs, just change from develop to `gci-dev` branch.
image

**Pull request merged**